### PR TITLE
Peer to peer code review

### PR DESCRIPTION
**Pull request by ksigei**
Does my coding partner follow JS best practices?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,14 @@
 
 # Project Name
```
Comment (Mo7amedSMahdi) [inline README.md:3]: - [ ] [Required] Kindly, Add the project name to the project readme.

File: README.md [hunk 2]
```diff
@@ -2,15 +2,14 @@
...
 <!-- ![screenshot](./images/screenshot.png)
 ![screenshot](./images/screenshot1.png)
 ![screenshot](./images/contact-scrn.png)
 ![screenshot](./images/footer-scrn.png)
 ![screenshot](./images/footer-scrn.png) -->
 
-
 ## Built With
 
 - HTML
```
Comment (Mo7amedSMahdi) [inline README.md:15]: - [ ] [Optional] Kindly, Add the full list of technology you used in this project for example ( HTML, JavaScript, React...etc).

File: dist/index.html [hunk 3]
```diff
@@ -27,7 +24,7 @@ <h1>Today's To Do</h1><span><i class="fas fa-sync-alt"></i></span>
            
         </div>
         <div class="button">
-            <button id="button">clear all completed</button>
+            <button id="button">Clear completed</button>
```
Comment (Mo7amedSMahdi) [inline dist/index.html:27]: - [ ] [Required] Kindly, Add button type for each button as it required by HTML [Webhint](https://webhint.io/) linter.

File: webpack.config.js [hunk 4]
```diff
@@ -8,7 +8,6 @@ module.exports = {
   },
   plugins: [
     new HtmlWebpackPlugin({
-      // title: 'Output Management',
```
Comment (Mo7amedSMahdi) [inline webpack.config.js:11]: - [ ] [OPTIONAL] Please, consider removing comments to make your code clear 👍

**Review by Mo7amedSMahdi — COMMENTED** [review]
# Required Changes ♻️

Hi, @ksigei,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Changes Required ♻️

- [ ] Check the comments under the review.

## Optional suggestions

- [ ] Please, consider my optional comments under the review before approval and merging 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: webpack.config.js (hunk 4 above)
Comment (ksigei) [reply webpack.config.js:11]: @Mo7amedSMahdi , thanks for the review. Let me work on the changes.

**Pull request merged**